gotocompany/compass/v1beta1/service.proto

```diff
@@ -1055,6 +1069,18 @@ message GetAllAssetsRequest {
   bool with_total = 10 [(grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
     description: "if set include total field in response"
   }];
+  string duration_before_current = 11 [
+    (validate.rules).string.ignore_empty = true,
+    (grpc.gateway.protoc_gen_openapiv2.options.openapiv2_field) = {
+      description: "filter by refreshed_at field < now - duration_before_current"
```

Comment:
where does the equality condition fit in?

Reply:
Currently, I don't including equality condition. Btw, that current name should be duration_for_upper_threshold.
What do you think about that? Is it better "<=" instead of "<"?